README.md

```diff
@@ -32,14 +32,31 @@ Set name of your newly created app to your environment to make things simpler:
 export HEROKU_APP_NAME=app_name  # please replace with your app name
 ```
 
-Finally run the following command and follow the instructions. You will be
-prompted to login to your Salesforce sandbox. 
+#### Configuring Salesforce connection
+
+You can configure a Salesforce connection using the UI:
+
+```bash
+heroku addons:open herokuconnect -a "$HEROKU_APP_NAME"
+```
+
+It will open the Heroku Connect page. Click on Setup and Next, Next, Next.
```

Comment:
>Click on Setup and Next, Next, Next.

Maybe change it to something like "Click on Setup and follow the process" or "follow form steps afterwards".

If you don't like it, feel free to like like it is. :)

Reply:
Sounds good, Rust. Thanks!